packages/react-devtools-shared/src/backend/console.js

```diff
@@ -209,7 +224,21 @@ export function patch({
           debugger;
         }
 
-        originalMethod(...args);
+        if (isInStrictMode) {
+          if (!consoleSettingsRef.hideDoubleLogsInStrictMode) {
+            // TODO(luna) pick a better color than gray
```

Comment:
I think this TODO can be removed, although I also think we may want to pick a color based on the method being logged.

For `log` and `trace` I think this color works fine.

For `warn` maybe `rgba(200, 150, 100, 0.5)`"
![Screen Shot 2021-08-06 at 2 31 08 PM](https://user-images.githubusercontent.com/29597/128556211-1421be48-a2ca-45e4-b9f2-ac24ce3bb84d.png)
![Screen Shot 2021-08-06 at 2 31 15 PM](https://user-images.githubusercontent.com/29597/128556212-6fc15706-c88a-4c8f-974f-652596168db9.png)

For `error` maybe `rgba(225, 100, 100, 0.5)`?
![Screen Shot 2021-08-06 at 2 31 33 PM](https://user-images.githubusercontent.com/29597/128556214-d6c0d82e-6291-487e-84bf-a0de92182883.png)
![Screen Shot 2021-08-06 at 2 31 38 PM](https://user-images.githubusercontent.com/29597/128556215-8a396082-6459-44ad-80f5-f52906eb7b9d.png)

Reply:
Edit: Looks like you've already changed this code while I was reviewing so feel free to ignore my color suggestions above 😄